From 7becb4ff5d5f81832124bc5f76c7c5be1889d772 Mon Sep 17 00:00:00 2001
From: Mike Hommey <mh@glandium.org>
Date: Tue, 30 Apr 2024 05:40:58 +0900
Subject: [PATCH] Use clang_getFileLocation instead of
 clang_getSpellingLocation

Both had the same behavior... until https://github.com/llvm/llvm-project/commit/2e770edd8ce13f48402f1d93e5fb982d8a2ebe64
which fixed getSpellingLocation, but bindgen looks like it actually expects
the getFileLocation result.

Note: Backported from bindgen v0.70.0
---
 clang.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/clang.rs
+++ b/clang.rs
@@ -1608,7 +1608,7 @@
             let mut line = 0;
             let mut col = 0;
             let mut off = 0;
-            clang_getSpellingLocation(
+            clang_getFileLocation(
                 self.x, &mut file, &mut line, &mut col, &mut off,
             );
             (File { x: file }, line as usize, col as usize, off as usize)
